devstack/Readme-NFP-install.txt

```diff
@@ -26,7 +26,7 @@ Fresh Installation Steps:
 
     * Enterprise Mode Configuration:
          # Devstack installation in enterprise mode
-             Example: [[ $ENABLE_NFP = True ]] && DEVSTACK_MODE=enterprise
```

Comment:
This Readme in our repo is for enterprise mode also.
But in the submitted Readme, we removed enterprise mode related lines.